### PR TITLE
Questions 1 to 132

### LINKED ISSUE #2: Update question 49
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/49/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/49/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `127386`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/49/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/49/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #5: Update question 35
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/35/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/35/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `12`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/35/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/35/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #6: Update question 26
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/26/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/26/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program has undefined behavior

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/26/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/26/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #25: Update question 52
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/52/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/52/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program has a compilation error

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/52/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/52/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #27: Update question 33
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/33/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/33/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `G`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/33/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/33/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #29: Update question 48
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/48/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/48/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `z`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/48/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/48/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #36: Update question 42
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/42/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/42/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `1133`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/42/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/42/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #37: Update question 41
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/41/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/41/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `B`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/41/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/41/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #43: Update question 38
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/38/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/38/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `11`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/38/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/38/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #50: Update question 17
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/17/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/17/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `abBA`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/17/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/17/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #52: Update question 44
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/44/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/44/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `YX`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/44/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/44/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #55: Update question 227
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/227/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/227/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `1`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/227/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/227/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #67: Update question 144
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/144/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/144/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `010`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/144/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/144/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #75: Update question 29
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/29/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/29/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `121`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/29/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/29/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #80: Update question 32
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/32/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/32/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `abbc`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/32/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/32/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #81: Update question 30
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/30/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/30/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: ``

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/30/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/30/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #87: Update question 15
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/15/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/15/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `acabBA`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/15/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/15/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #93: Update question 31
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/31/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/31/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program has a compilation error

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/31/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/31/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### LINKED ISSUE #97: Update question 131
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/131/) from C++11 to C++17.

You'll find the source code in [question.cpp](../blob/master/questions/131/question.cpp), this should normally not need modification.

In C++11, the correct answer is:
> The program is guaranteed to output: `id`

Please verify that this is still correct in C++17. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/131/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/131/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++11 to C++17, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### PR DESCRIPTION
**Pull request by Rakete1111**
There are some merge conflicts (because those questions were already fixed) that you can safely ignore, <s>although my git skills are pretty bad so I don't know how to do it</s> nvm figured it out . I went through Q1 to 132 and fixed all of those that were needed.

Hope I didn't forget some issues

Fixes #63
Fixes #73
Fixes #83
Fixes #60
Fixes #90
Fixes #10
Fixes #31
Fixes #32
Fixes #49
Fixes #28
Fixes #57
Fixes #48
Fixes #94
Fixes #61
Fixes #103
Fixes #70
Fixes #41
Fixes #26
Fixes #18
Fixes #7
Fixes #45
Fixes #51
Fixes #53
Fixes #65
Fixes #6
Fixes #22
Fixes #75
Fixes #81
Fixes #93
Fixes #80
Fixes #27
Fixes #5
Fixes #54
Fixes #43
Fixes #37
Fixes #36
Fixes #54
Fixes #43
Fixes #37
Fixes #36
Fixes #52
Fixes #29
Fixes #2
Fixes #25
Fixes #87 
Fixes #50
Fixes #55

### CODE REVIEW

**Comment by Rakete1111** [comment]
Wow, thanks for the list. You're right, the questions that have no update don't reference the standard. The right thing to do is to close them, right?

**Comment by knatten** [comment]
Don't thank me, thank vim, sed, awk, grep and bash! ;)

I checked the questions that had no fixes, and they do indeed lack references to the standard, with two exceptions.
- Question 131 (Issue 97) This one is already closed, you probably had your own fix before you fixed the merge conflict
- Question 144 (issue 67) Maybe just a typo

So I suggest you update the text in your PR adding "Fixes #55" and removing "Fixes #67" and "Fixes #97". 

Then I'll review all your changes.

**Comment by knatten** [comment]
And yes, it's good to leave in your "Fixes #<issue>" lines for the ones that don't need updating, so they'll get automatically closed when I merge this PR.

**Comment by Rakete1111** [comment]
I wish I could use those tools...

Ok great done.

File: questions/127/explanation.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-[expr.prim.lambda]§5.1.2¶18 says
+[expr.prim.lambda]§8.5.1¶18 says
```
Comment (knatten) [inline questions/127/explanation.md]: This paragraph doesn't exist, at least not in my copy of the standard. The correct one seems to be [expr.prim.lambda.capture]§8.1.5.2¶14

(Note that I only have access to the last draft, not the officially released C++17 standard. I bought the C++11 standard back in the day, I guess I need to buy the C++17 one too.)

File: questions/42/explanation.md [hunk 2]
```diff
@@ -1,20 +1,11 @@
...
-— If the initializer list has no elements and T is a class type with a default constructor, the object is value-initialized.
-— Otherwise, if T is an aggregate, aggregate initialization is performed (§8.5.1).
-— Otherwise, if T is a specialization of std::initializer_list<E>, an initializer_list object is constructed as described below and used to initialize the object according to the rules for initialization of an object from a class of the same type (§8.5).
-— Otherwise, if T is a class type, constructors are considered. The applicable constructors are enumerated and the best one is chosen through overload resolution (§13.3, §13.3.1.7). If a narrowing conversion (see below) is required to convert any of the arguments, the program is ill-formed.
-— (more cases...)"
-
-`a1` is default initialized, as described in [dcl.init.list]§8.5.¶11
+`a1` is default initialized, as described in [dcl.init]§11.6¶12.
 
 `a2` doesn't actually use the initializer_list constructor with a list of zero elements, but the default constructor, as described by the first option of the list above.
```
[the remark this answers is not in the document]
Comment (Rakete1111) [reply questions/42/explanation.md]: The list above never mentioned default initialization, and I dont even think that it adds meaningful information. That's why I deleted it.

File: questions/132/explanation.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
-Is `foo` called both times or just once? The C++11 standard says this in [dcl.fct.default]§8.3.6¶9: "Default arguments are evaluated each time the function is called."
+Is `foo` called both times or just once? The C++11 standard says this in [dcl.fct.default]§11.3.6¶9: "A default argument is evaluated each time the function is called with no argument for the corresponding parameter."
```
[the remark this answers is not in the document]
Comment (Rakete1111) [reply questions/132/explanation.md]: Right. Will fix.

File: questions/127/explanation.md (hunk 1 above)
Comment (Rakete1111) [reply questions/127/explanation.md]: You are right.

Also (C++17 draft): https://timsong-cpp.github.io/cppwp/n4659/

**Comment by TartanLlama** [comment]
image

File: questions/42/explanation.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Rakete1111) [reply questions/42/explanation.md]: Ah, I see. Will fix.

**Pull request merged**